sympy/sets/sets.py

```diff
@@ -278,10 +278,10 @@ def contains(self, other):
 
         """
         other = sympify(other, strict=True)
-        ret = self._contains(other)
+        ret = sympify(self._contains(other))
```

Comment:
This is the error is get:
```
>>> (S.Complexes).union(S.Reals)

^CTraceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/gxyd/Public/sympy/sympy/sets/sets.py", line 85, in union
    return Union(self, other)
  File "/home/gxyd/Public/sympy/sympy/sets/sets.py", line 1182, in __new__
    return Union.reduce(args)
  File "/home/gxyd/Public/sympy/sympy/sets/sets.py", line 1216, in reduce
    new_set = s._union(t)
  File "/home/gxyd/Public/sympy/sympy/sets/sets.py", line 954, in _union
    if ((self.left_open and other.contains(self.start) is true) or
  File "/home/gxyd/Public/sympy/sympy/sets/sets.py", line 282, in contains
    print('ret', ret, self, other)
  File "/home/gxyd/Public/sympy/sympy/core/basic.py", line 397, in __str__
    return sstr(self, order=None)
  File "/home/gxyd/Public/sympy/sympy/printing/str.py", line 742, in sstr
    s = p.doprint(expr)
  File "/home/gxyd/Public/sympy/sympy/printing/printer.py", line 233, in doprint
    return self._str(self._print(expr))
  File "/home/gxyd/Public/sympy/sympy/printing/printer.py", line 257, in _print
    return getattr(self, printmethod)(expr, *args, **kwargs)
  File "/home/gxyd/Public/sympy/sympy/printing/str.py", line 89, in _print_Basic
    l = [self._print(o) for o in expr.args]
  File "/home/gxyd/Public/sympy/sympy/printing/str.py", line 89, in <listcomp>
    l = [self._print(o) for o in expr.args]
  File "/home/gxyd/Public/sympy/sympy/printing/printer.py", line 257, in _print
    return getattr(self, printmethod)(expr, *args, **kwargs)
  File "/home/gxyd/Public/sympy/sympy/printing/str.py", line 198, in _print_Lambda
    return "Lambda((%s), %s)" % (arg_string, expr)
  File "/home/gxyd/Public/sympy/sympy/core/basic.py", line 397, in __str__
    return sstr(self, order=None)
  File "/home/gxyd/Public/sympy/sympy/printing/str.py", line 742, in sstr
    s = p.doprint(expr)
  File "/home/gxyd/Public/sympy/sympy/printing/printer.py", line 233, in doprint
    return self._str(self._print(expr))
  File "/home/gxyd/Public/sympy/sympy/printing/printer.py", line 235, in _print
    def _print(self, expr, *args, **kwargs):
KeyboardInterrupt
```
I introduced `sympify` here and this bug occurs.
I could not understand why is this happening?